boards/FSM/Inc/Io/Io_MSP3002K5P3N1.h

```diff
@@ -1,5 +1,12 @@
 #pragma once
 
+/**
+ * Get the brake actuation status from the MSP-300-2K5-P-3-N-1 brake pressure
+ * sensor
+ * @return True if the brake is actuated, false if the brake is not actuated
+ */
```

Comment:
The description is misleading, the threshold output doesn't come directly from the brake pressure sensor
```suggestion
/**
 * Check if the pressure from MSP-300-2K5-P-3-N-1 exceeds the brake pressure threshold. If the pressure exceeds threshold, the brake is said to be actuated.
 * @return True if the brake is actuated, false if the brake is not actuated
 */
```